add note to cloud sso faq (#15335)
cockroachdb/docs: cockroachcloud/cloud-sso.md

### Which authentication flows are supported with Enterprise Authentication? Is it possible to enable the identity provider initiated flow?

> The primary flow is to initiate {{ site.data.products.db }} SSO through the cloud console UI.

Reviewer: ```suggestion
The primary flow is to initiate {{ site.data.products.db }} SSO through the Cockroach Cloud Console.
```

> Initiation from the IdP is also possible, but not supported through the cloud console. Contact your account team for more information.

Reviewer: I wonder if it makes sense to rephrase this as "If you need to initiate SSO integration from the IdP, contact your account team for assistance."

> Initiation from the IdP is also possible, but not supported through the cloud console. Contact your account team for more information.

Author: ```suggestion
If you need to initiate SSO integration from the IdP, contact your account team for assistance
```

### Which authentication flows are supported with Enterprise Authentication? Is it possible to enable the identity provider initiated flow?

The primary flow is to initiate {{ site.data.products.db }} SSO through the Cockroach Cloud Console.

> If you need to initiate SSO integration from the IdP, contact your account team for assistance

Reviewer: ```suggestion
If you need to initiate SSO integration from the IdP, contact your account team for assistance.
```
